shopelectro/views/catalog.py

```diff
@@ -18,16 +18,11 @@
 from shopelectro.exception import Http400
 from shopelectro.views.helpers import set_csrf_cookie
 
-
 # block numeric indexes to limit
 MATRIX_BLOCKS_TO_LIMIT = [3, 5]
 MATRIX_BLOCK_SIZE = 7
 
 
-# @todo #822:60m  Create tests for the catalog matrix.
-#  This cases are would be good to test:
-#  - Matrix sorting by page position
-#  - Block items limiting
```

Comment:
> Block items limiting

It seems you still haven't test this

Reply:
y, did it intentionally. Updated PR body about it